src/components/disclaimer-modal/disclaimer-modal.module.css

```
@@ -8,20 +8,34 @@
   width: 300px;
-  top: 50px;
+  max-width: calc(100% - 20px);
+  max-height: calc(100vh - 40px);
+  overflow-y: auto;
+  top: 50%;
   left: 50%;
-  margin-left: -151px;
+  transform: translate(-50%, -50%);
   background-color: var(--disclaimer-modal-background-color, rgb(255, 255, 255));
   border: var(--disclaimer-modal-border, 1px solid rgb(136, 0, 0));
   color: var(--disclaimer-modal-text-color, rgb(136, 0, 0));
```

Stylelint (17.12.0)

[error] Duplicate property "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)